### PR TITLE
Feature/UI improved blockly

### PR DESCRIPTION
**Pull request by an-desco**
Fixing bugs

### CODE REVIEW

File: client/src/routes/automation/Blockly.svelte [hunk 1]
```diff
@@ -10,22 +10,17 @@
...
-            <block type="binary_condition" />
         </category>
-        <category name="Actions" colour={COLORS.HUE_ORANGE}>
-            <category name="Services" colour={COLORS.HUE_ORANGE}>
-                <block type="action" />
-                <!-- <block type="services"></block> -->
+        <category name="Actions" colour={COLORS.HUE_RED}>
+            <category name="Services" colour={COLORS.HUE_RED}>
+                <block type="objet_action" />
+                <!-- <block type="action" /> -->
```
Comment (Nailykk) [inline client/src/routes/automation/Blockly.svelte]: Is this comment relevant ?

File: client/src/routes/automation/BlocklyComponent.svelte [hunk 2]
```diff
@@ -39,7 +72,7 @@
 
 <style scoped>
     #blocklyDiv {
-        height: 600px;
+        height: 700px;
```
Comment (Nailykk) [inline client/src/routes/automation/BlocklyComponent.svelte:75]: Is there a way to use relative values instead ?

File: client/src/routes/automation/esidom_generator.ts [hunk 3]
```diff
@@ -86,7 +86,8 @@ interface BlocksGenerator {
             json.action = JSON.parse(actions);
         }
 
-        json.dropdown_mode = dropdown_mode;
+        // json.dropdown_mode = dropdown_mode;
+        json.dropdown_mode = 'single';
```
Comment (Nailykk) [inline client/src/routes/automation/esidom_generator.ts:94]: A comment may be useful to understand why it's hard coded ?

**Review by Nailykk — CHANGES_REQUESTED** [review]

File: client/src/routes/automation/BlocklyComponent.svelte [hunk 4]
```diff
@@ -11,9 +12,41 @@
         const toolbox: HTMLElement | undefined =
```
Comment (asoran) [inline client/src/routes/automation/BlocklyComponent.svelte:12]: Why is it typed with ` | undefined` ?

**Review by asoran — APPROVED** [review]
:see_no_evil:

File: client/src/routes/automation/BlocklyComponent.svelte (hunk 2 above)
Comment (an-desco) [reply client/src/routes/automation/BlocklyComponent.svelte:75]: In the next version, when I will add my tailwind implementation

File: client/src/routes/automation/Blockly.svelte (hunk 1 above)
Comment (an-desco) [reply client/src/routes/automation/Blockly.svelte]: comment and block declaration erased

File: client/src/routes/automation/BlocklyComponent.svelte (hunk 4 above)
Comment (an-desco) [reply client/src/routes/automation/BlocklyComponent.svelte:12]: erased

File: client/src/routes/automation/esidom_generator.ts (hunk 3 above)
Comment (an-desco) [reply client/src/routes/automation/esidom_generator.ts:94]: Yes sir

**Review by Nailykk — APPROVED** [review]

**Pull request merged**